packages/svelte/test/js/samples/hoisted-const/input.svelte

```diff
@@ -1,6 +1,7 @@
 <script>
 	const ANSWER = 42;
+	const { length } = 'abc';
```

Comment:
is there a reason to call this `length`? i'd expect a variable with that name to be a `number`

Reply:
The destructured property `length` of the string is a number.

Reply:
Oh, yeah 😆 It's so rare that I see this used on a primitive and not a plain object that I didn't parse it the first time around